R/utils-numeric.R

```diff
@@ -27,7 +27,7 @@ getDiscretizedBins <- function(x, method = c('equalInterval', 'quantile', 'sd'),
   }
 
   if (isDate) {
-    binEdges <- as.Date(binEdges, origin = "1900-01-01") # this is the default origin
+    binEdges <- as.Date(binEdges, origin = "1970-01-01")
```

Comment:
lol. i still think this needs the explanation in the comment though. the idea is to make sure anyone reading it later realizes this needs to use the same origin as the call to as.numeric earlier..